nearcore/src/config.rs

```diff
@@ -822,13 +822,21 @@ pub fn init_configs(
     download_config_url: Option<&str>,
     boot_nodes: Option<&str>,
     max_gas_burnt_view: Option<Gas>,
-) {
+) -> Result<()> {
+    info!("creating directory at {:?}", dir);
```

Comment:
@matklad Well, not anymore. Now, we actually print the folder name in case of failure, and that's what it was needed for.

```
    fs::create_dir_all(dir).with_context(|| anyhow!("Failed to create directory {:?}", dir))?;
```